src/main/resources/openapi/transcriptions.yaml

```diff
@@ -0,0 +1,63 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /transcriptions:
+    post:
+      tags:
+        - Transcription
+      summary: Users can request transcription
+      operationId: requestTranscription
+      description: Adds a user transcription request for processing
+      responses:
+        '200':
+          description: OK
+        '404':
+          description: Resource with the provided id does not exist.
+        '500':
```

Comment:
This can go - if you change the TranscriptionError validation to be 400.
I wouldn't really expect to see 500 unless something bad happens, unexpectedly